gammapy/makers/background/reflected.py

```diff
@@ -451,6 +451,9 @@ def __init__(
         if exclusion_mask and not exclusion_mask.is_mask:
             raise ValueError("Exclusion mask must contain boolean values")
 
+        if exclusion_mask and not exclusion_mask.geom.is_image:
```

Comment:
```suggestion
        if exclusion_mask and not exclusion_mask.geom.is_flat:
```
this will work with 1D axis too.

Reply:
Maybe log a warning if flat but not image to say that the additional axis edges are not taken into account.